charts/camunda-platform-alpha/values.yaml

```diff
@@ -77,6 +77,12 @@ global:
     ## @param global.multitenancy.enabled if true, then enable multitenancy in all applicable components.
     enabled: false
 
+  ## Authorizations configuration.
+  ## @extra global.authorizations
+  authorizations:
+    ## @param global.authorizations.enabled if true, then enable authorizations checks in all applicable components.
+    enabled: false
```

Comment:
Any reason to have it disabled by default? We have enabled it in previous versions.

Reply:
it is specified in this [doc](https://docs.google.com/document/d/1ikxw9ARCv-MBwASnj7WhW5uHCndCRk2B35xi-cB2N0M/edit?tab=t.0#heading=h.jzwosqivdg8k)
>Authorization can be enabled or disabled per C8 cluster by configuration properties, default: authorizations are disabled.

@Ben-Sheppard @FarkasRabai can you confirm that?

Reply:
The defaults in the app are different from those in Helm.

It's the same for older versions of Camunda, like 8.6. In the app config, the auth is disabled, but it's enabled in the Helm chart as it's meant for use in non-local environments.

Reply:
@aabouzaid can you point where authorizations are enabled in 8.6 ?
maybe you are referring to `global.identity.auth.enabled`? this enables auth**entitication** with identity and not authorizations

Reply:
@houssain-barouni Ah, I misread it as `authentication`. Thanks for the clarification :ok_hand: 

Cool, then we can move forward :+1: